### PR TITLE
Feat/update participation section about page

### PR DESCRIPTION
**Pull request by YouQam**
This PR includes updating participation section texts and colors. Also, adding send email functionality when clicking on one of the boxes in the mentioned section.

### CODE REVIEW

File: src/app/public/components/about/about.component.html [hunk 1]
```diff
@@ -157,27 +157,30 @@ <h3>Klima Campus</h3>
...
-						<h2 class="h2-bold" style="color: #ffffff">Erweiterung des Plugins fördern</h2>
-					</div>
+					<a class="join-us-card tw-w-[30%] tw-bg-purple" href='mailto:{{mailAdrees}}?subject={{mailBody}}'>
+						<h2 class="h2-bold !tw-text-white">OEB fördern</h2>
+					</a>
+					<a class="join-us-card tw-w-[40%] tw-bg-yellow" href='mailto:{{mailAdrees}}?subject={{mailBody}}'>
+						<h2 class="h2-bold">Aktiv mitmachen bei Open Educational Badges</h2>
+					</a>
+					<a class="join-us-card tw-w-[30%] tw-bg-red" href='mailto:{{mailAdrees}}?subject={{mailBody}}'>
+						<h2 class="h2-bold !tw-text-white">Ein individuelles PlugIn anfragen</h2>
```
Comment (timber-they) [inline src/app/public/components/about/about.component.html:182]: I'd typically write it *Plugin* and not *PlugIn*

**Review by timber-they — APPROVED** [review]

**Review by timber-they — CHANGES_REQUESTED** [review]
Ah I accidently approved it, wanted to request changes

**Comment by timber-they** [comment]
Also, is it on purpose that all the buttons do exactly the same thing?

File: src/app/public/components/about/about.component.html (hunk 1 above)
Comment (YouQam) [reply src/app/public/components/about/about.component.html:182]: We can suggest this to Mycelia team ;)

**Comment by YouQam** [comment]
> Also, is it on purpose that all the buttons do exactly the same thing?

atm, yes.

File: src/app/public/components/about/about.component.html (hunk 1 above)
Comment (YouQam) [reply src/app/public/components/about/about.component.html:182]: I've already contacted them to choose between `plug-In` or `plugin`

Comment (YouQam) [reply src/app/public/components/about/about.component.html]: The change was resolved.

File: src/app/public/components/about/about.component.html [hunk 2]
```diff
@@ -157,27 +157,30 @@ <h3>Klima Campus</h3>
...
-						<h2 class="h2-bold" style="color: #ffffff">Erweiterung des Plugins fördern</h2>
-					</div>
+					<a class="join-us-card tw-w-[30%] tw-bg-purple" href='mailto:{{mailAdrees}}?subject={{mailBody}}'>
+						<h2 class="h2-bold !tw-text-white">OEB fördern</h2>
+					</a>
+					<a class="join-us-card tw-w-[40%] tw-bg-yellow" href='mailto:{{mailAdrees}}?subject={{mailBody}}'>
+						<h2 class="h2-bold">Aktiv mitmachen bei Open Educational Badges</h2>
+					</a>
+					<a class="join-us-card tw-w-[30%] tw-bg-red" href='mailto:{{mailAdrees}}?subject={{mailBody}}'>
+						<h2 class="h2-bold !tw-text-white">Ein individuelles Plug-in anfragen</h2>
```
Comment (jona159) [inline src/app/public/components/about/about.component.html:182]: If the ```!``` is really needed, please add a comment why

File: src/app/public/components/about/about.component.ts [hunk 3]
```diff
@@ -6,7 +6,10 @@ import { Component, OnInit } from '@angular/core';
 	styleUrls: ['./about.component.css'],
 })
 export class AboutComponent implements OnInit {
-	constructor() {}
+  mailAdrees = "hallo@openbadges.education";
```
[the remark this answers is not in the document]
Comment (YouQam) [reply src/app/public/components/about/about.component.ts]: I renamed it to mailAddress.

File: src/app/public/components/about/about.component.html (hunk 2 above)
Comment (YouQam) [reply src/app/public/components/about/about.component.html:184]: I added comments.

**Review by jona159 — APPROVED** [review]

**Review by timber-they — APPROVED** [review]

**Pull request merged**